troubleshooting/debugging.md

```diff
@@ -816,11 +816,11 @@ When you recieve a SIGSEGV/crash report from a FireLens customer, perform the fo
 
 ##### 1. Build and distribute a core dump S3 uploader image
 
-You need a customized image build for the specific version/case you are testing. Make sure the `ENV FLB_VERSION` is set to the right version in the `Dockerfile.debug-base` and make sure the `AWS_FLB_CHERRY_PICKS` file has the right contents for the release you are testing. 
+You need a customized image build for the specific version/case you are testing. Make sure the `ENV FLB_VERSION` is set to the right version in the `scripts/dockerfiles/Dockerfile.build` and make sure the `AWS_FLB_CHERRY_PICKS` file has the right contents for the release you are testing. 
 
 Then simply run:
 ```
-make core
+make debug
```

Comment:
Let's give the names/tags of the outputted images?

Reply:
Sure!

Reply:
See line 836, below

Reply:
The resulting image will be tagged `amazon/aws-for-fluent-bit:debug` and `amazon/aws-for-fluent-bit:debug-s3`.